js/scorm/cookieLMS.js

```diff
@@ -0,0 +1,187 @@
+import Adapt from 'core/js/adapt';
+import Cookies from 'libraries/js-cookie.js';
+
+/** Start the mock API if window.ISCOOKIELMS exists and isn't null */
+export const shouldStart = (Object.prototype.hasOwnProperty.call(window, 'ISCOOKIELMS') && window.ISCOOKIELMS !== null);
+
+/** Store the data in a cookie if window.ISCOOKIELMS is true, otherwise setup the API without storing data. */
+export const isStoringData = (window.ISCOOKIELMS === true);
+
+export function createResetButton() {
+  $('body').append($('<style id="spoor-clear-button">.spoor-reset-button { position:fixed; right:0px; bottom:0px; } </style>'));
```

Comment:
I think these two parts of the code could be put into variables, e.g.

`const resetButtonStyle = '<style id="spoor-clear-button">.spoor-reset-button { position:fixed; right:0px; bottom:0px; } </style>';`
`const resetButton = '<button class="spoor-reset-button btn-text">Reset</button>'`

and then both of these appended to the body element in one go instead of appending to the body element twice on separate lines of code